src/queueing_honey_badger.rs

```diff
@@ -122,6 +122,8 @@ where
     output: VecDeque<Batch<Tx, NodeUid>>,
 }
 
+pub type QueueingHoneyBadgerStep<Tx, NodeUid> = Step<NodeUid, Batch<Tx, NodeUid>>;
```

Comment:
`<N>` vs. `<NodeUid>`

Reply:
I plan to reorganize the traits, adding supertraits...
```
    /// A transaction, user message, etc.
    pub trait Contribution: Eq + Debug + Hash {}
    impl<C> Contribution for C where C: Eq + Debug + Hash {}

    /// A peer node's unique identifier.
    pub trait NodeUid: Eq + Ord + Clone + Debug + Hash {}
    impl<N> NodeUid for N where N: Eq + Ord + Clone + Debug + Hash  {}
````
and shortening the trait parameters to single letters sometime when the churn dies down a bit.

Reply:
@c0gent, I used long names but please feel free to promote single-letter type variables if you have to.

Reply:
I think the usual style is single-letter? At least that is what I am used to from the stdlib and other crates.

Reply:
Yeah I reorganize/rename type params. per: https://github.com/poanetwork/hbbft/issues/143 probably next week when I have time.